external/llvm-project/mlir/lib/ExecutionEngine/CMakeLists.txt

```diff
@@ -208,7 +207,7 @@ if(MLIR_ENABLE_ROCM_RUNNER)
 
   # Locate HIP runtime library.
   find_library(ROCM_RUNTIME_LIBRARY amdhip64
-               PATHS "${HIP_PATH}/lib")
+               PATHS "${HIP_ROOT_DIR}/lib")
```

Comment:
This is changed in the current miopen-dialect branch. I cannot see when it happened since github blame does not work now.

Reply:
Ah, right - I changed the other way in the subtree update and now am moving it back